src/perf/lib/PerfServer.cpp

```diff
@@ -71,6 +73,48 @@ PerfServer::Init(
         }
     }
 
+    if (TryGetVariableUnitValue(argc, argv, "delay", &DelayMicroseconds, nullptr)) {
```

Comment:
```suggestion
    if (TryGetVariableUnitValue(argc, argv, "delay", &DelayMicroseconds, nullptr) && DelayMicroseconds != 0) {
```
Maybe ignore values of zero?